spec/order_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'order'
+require 'menu'
+
+describe Order do
+  subject(:order) { described_class.new(menu) }
+  let(:menu) { instance_double('Menu') }
+  let(:dish) { {'Classic Fondue': 2,'Special Fondue': 1} }
+  # let(:price) { {'Classic Fondue': 13.0, 'Special Fondue': 14.5} }
+
+  describe '#add' do
+
+    before do
+      allow(menu).to receive(:available?).with(:'Classic Fondue').and_return(true)
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)